src/components/PropertyOwner/PropertyOwnerContent.tsx

```diff
@@ -25,16 +26,33 @@ export function PropertyOwnerContent({ uri, hideSubowners = false }: Props) {
     isGlobeLayersUri(uri, state.properties.properties)
   );
 
-  const visibleProperties = useVisibleProperties(propertyOwner);
+  let visibleProperties = useVisibleProperties(propertyOwner);
   const subowners = propertyOwner.subowners ?? [];
 
+  // Separate video player subowner if it exists, so we can render a custom component.
+  // Note that we don't want to do this if the current property owner is the TileProvider
+  // since in that case the VideoPlayer would be rendered at the GlobeLayer level.
+  const videoPlayerUri = uri.endsWith('.TileProvider')
+    ? undefined
+    : subowners.find((subowner) => subowner.endsWith('.VideoPlayer'));
+
+  // Special handling to hide certain properties from the VideoPlayer property owner,
+  // since we render them in a custom way in the VideoPlayerComponent
+  if (propertyOwner.identifier === 'VideoPlayer') {
+    const PropertiesToHide = ['.Play', '.Pause', '.GoToStart'];
+    visibleProperties = visibleProperties.filter(
+      (propUri) => !PropertiesToHide.some((s) => propUri.endsWith(s))
```

Comment:
I don't remember from the meeting, but why are we not hiding the Loop and Audio checkboxes?

Reply:
My idea here was to keep "setting-like" things in the property owner menu. And these two things felt like settings to me